Makefile

```diff
@@ -258,6 +259,14 @@ test-go:
 test-python:
 	@./build/test.sh python
 
+test-e2e-existing-clusters:
+	@eval $$(python3 ./manager/cluster_config_env.py ./dev/config/cluster-aws.yaml) && CORTEX_CLI_PATH=./bin/cortex ./build/test.sh apis -p aws -e "$$CORTEX_CLUSTER_NAME-aws"
+	@eval $$(python3 ./manager/cluster_config_env.py ./dev/config/cluster-gcp.yaml) && CORTEX_CLI_PATH=./bin/cortex ./build/test.sh apis -p gcp -e "$$CORTEX_CLUSTER_NAME-gcp"
+test-e2e-existing-cluster-aws:
+	@eval $$(python3 ./manager/cluster_config_env.py ./dev/config/cluster-aws.yaml) && CORTEX_CLI_PATH=./bin/cortex ./build/test.sh apis -p aws -e "$$CORTEX_CLUSTER_NAME-aws"
+test-e2e-existing-cluster-gcp:
+	@eval $$(python3 ./manager/cluster_config_env.py ./dev/config/cluster-gcp.yaml) && CORTEX_CLI_PATH=./bin/cortex ./build/test.sh apis -p gcp -e "$$CORTEX_CLUSTER_NAME-gcp"
```

Comment:
nit: command naming is a bit too long

Reply:
Not too concerned about this - there are commands longer than this in Makefile.

Reply:
We want to go with `test-e2e-aws/gcp` for existing clusters and `test-e2e-new-aws/gcp` for new clusters to be spun up.